src/PlusDataCollection/CMakeLists.txt

```diff
@@ -1194,6 +1195,57 @@ IF(PLUS_USE_CAPISTRANO_VIDEO)
 ENDIF()
 
 # --------------------------------------------------------------------------
+# WinProbe support
+IF(PLUS_USE_WINPROBE_VIDEO)
+  SET(WinProbe_SRCS
+    WinProbe/vtkPlusWinProbeVideoSource.cxx
+    )
+  IF(MSVC OR ${CMAKE_GENERATOR} MATCHES "Xcode")
+    SET(WinProbe_HDRS
+      WinProbe/vtkPlusWinProbeVideoSource.h
+      )
+  ENDIF()
+  
+  SET(PlusDataCollection_HDRS ${PlusDataCollection_HDRS}
+    ${WinProbe_HDRS}
+    )
+
+  SET(PlusDataCollection_SRCS ${PlusDataCollection_SRCS}
+    ${WinProbe_SRCS}
+    )
+
+  SET(PlusDataCollection_INCLUDE_DIRS ${PlusDataCollection_INCLUDE_DIRS}     
+    ${WINPROBESDK_DIR}
+    ${CMAKE_CURRENT_SOURCE_DIR}/WinProbe
+    )
+
+  FILE(GLOB WinProbeXMLFiles "${WINPROBESDK_DIR}/xml/*.xml")
+  FILE(COPY ${WinProbeXMLFiles} DESTINATION ${CMAKE_BINARY_DIR})
```

Comment:
If I copy the XML files there, running CTest from command line fails. And working directory path gets modified somehow ([commit](https://github.com/PlusToolkit/PlusLib/pull/51/commits/c363a2a1e798120596a95a13998e368e799b79cd)), so I couldn't just rely on WORKING_DIRECTORY argument for the test.